src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -80,6 +83,9 @@ const LibrariesTodayHours = ({libraries, ...props}: { libraries: HoursProps["lib
               {library?.suLibraryHours &&
                 <TodayLibraryHours branchId={library.suLibraryHours}/>
               }
+              {library?.map &&
+                <span className="float-right w-auto inline"><Link href={library?.map} aria-label="Link to map"><MapPinIcon title="Map" width={25} className="mr-5 inline"/>Location</Link></span>
```

Comment:
I feel like some `sr-only` descriptive text would be helpful here. like `<span className="sr-only">{library.title}</span> Location`

Reply:
I think I put this in the right place...